io.catenax.early_warning_notification/1.0.0/EarlyWarningNotification.ttl

```diff
@@ -0,0 +1,153 @@
+   samm:characteristic :UniqueIdentifiersCharacteristic .
+
+:PopulationFilterCharateristic a samm-c:List ;
+   samm:preferredName "Population Filter Charateristic"@en ;
+   samm:description "Characteristic for the filter"@en ;
+   samm:dataType :PopulationFilter .
+
+:UniqueIdentifiersCharacteristic a samm-c:List ;
+   samm:preferredName "Unique Identifiers Characteristic"@en ;
+   samm:description "Characteristic for the UIDs"@en ;
+   samm:dataType :UniqueIdentifier .
+
+:PopulationFilter a samm:Entity ;
+   samm:preferredName "Population Filter"@en ;
+   samm:description "Population filter for the early warning data"@en ;
+   samm:properties ( :aspectProperty :aspectModel :rangeFrom :rangeTo :dataType :values ) .
+
+:UniqueIdentifier a samm:Entity ;
+   samm:preferredName "Unique Identifier"@en ;
+   samm:description "A UID"@en ;
+   samm:properties ( :anonymizedVIN :catenaXId :serialPartNumber ) .
+
+:aspectProperty a samm:Property ;
+   samm:preferredName "Aspect Property"@en ;
+   samm:description "A property from the aspect model referenced by \"aspectModel\""@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "production.productionDate" .
+
+:aspectModel a samm:Property ;
+   samm:preferredName "Aspect Model"@en ;
+   samm:description "A reference to an aspect model standardized by Catena-X for the quality use case"@en ;
+   samm:characteristic samm-c:ResourcePath ;
+   samm:exampleValue "urn:bamm:io.catenax.manufactured_parts_quality_information:1.0.0"^^xsd:anyURI .
+
+:rangeFrom a samm:Property ;
+   samm:preferredName "ValueRange_From"@en ;
+   samm:description "Start value of a value range for an aspect property "@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2019-02-01T00:00:00" .
+
+:rangeTo a samm:Property ;
+   samm:preferredName "ValueRange_To"@en ;
+   samm:description "End value of a value range for an aspect property "@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "2019-07-30T23:59:59" .
+
+:dataType a samm:Property ;
+   samm:preferredName "Data Type"@en ;
+   samm:description "Data type for the aspect propoerty"@en ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "date" .
+
+:values a samm:Property ;
+   samm:preferredName "List of Values"@en ;
+   samm:description "A list of values for the aspect property"@en ;
+   samm:characteristic :ValuesCharacteristic .
+
+:anonymizedVIN a samm:Property ;
+   samm:preferredName "VAN"@en ;
+   samm:description "A unique but anonymized identifier for a vehicle"@en ;
+   samm:see <urn:samm:io.catenax.vehicle.product_description:3.0.0> ;
+   samm:characteristic samm-c:Text ;
+   samm:exampleValue "3747429FGH382923974682" .
+
+:catenaXId a samm:Property ;
+   samm:preferredName "Catena-X ID"@en ;
+   samm:description "A unique identifier for a part or a vehicle. The identifier relates to a Catena-X digital twin"@en ;
+   samm:characteristic samm-c:Text ;

```

Comment:
you can make use of shared aspect model (io.catenax.shared.uuid:1.0.0) for UUID.
you can refer this https://github.com/eclipse-tractusx/sldt-semantic-models/blob/a07bdd5b1515767d4f8d07deb47cf2a4aa728bef/io.catenax.quality_message_content/1.0.0/QualityMessageContent.ttl#L35C9-L35C17

Reply:
@Timothy-Ra please check this also.